jobs/config.json

```diff
@@ -11606,7 +11606,7 @@
       "--node-test-args=--feature-gates=DynamicKubeletConfig=true,PodPriority=true,LocalStorageCapacityIsolation=true --kubelet-flags=\"--cgroups-per-qos=true --cgroup-root=/\"",
       "--node-tests=true",
       "--provider=gce",
-      "--test_args=--nodes=1 --focus=\"\\[Serial\\]|\\[NodeFeature:.+\\]\" --skip=\"\\[Flaky\\]\"",
```

Comment:
What I actually wanted to do is to enforce `[Serial]` `AND` `[NodeFeature:.+]`.
Is there an easy way to express that? /cc @krzyzacy

Reply:
regex is hard :-)

Reply:
Does calling `--focus` twice work (I'd assume not...)?

Reply:
I don't think so..

Reply:
from stackoverflow: https://regex101.com/r/iS5jJ3/1, but not sure what golang takes

Reply:
/shrug

I'll fix the test job with this PR first, and look for a way to express this better later